implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -11,14 +11,20 @@ use ockam::remote::RemoteForwarder;
 use ockam::{Address, Context, Result, Route, Routed, TcpTransport, Worker};
 use ockam_core::compat::{boxed::Box, collections::BTreeMap, string::String};
 use ockam_core::errcode::{Kind, Origin};
-use ockam_identity::authenticated_storage::mem::InMemoryStorage;
 use ockam_identity::{Identity, TrustEveryonePolicy};
 use ockam_multiaddr::MultiAddr;
 use ockam_vault::Vault;
 
+use crate::auth::Server;
+use crate::identity::IdentityService;
+use crate::lmdb::LmdbStorage;
```

Comment:
Nice observation. I think it's fine since lmdb feature is enabled by default and we don't want to support other types of storages for now

Reply:
It would be unfortunate if the crate does not compile with `default-features = false` so I recommend to either remove the "lmdb" feature and always make LMDB storage available or to conditionally expose it.

Reply:
I think conditionally exposing it is the way to go, but doesn't make sense to me until we have some viable alternative storage